### ISSUE TITLE
Telemetry not working on SP Racing F7 Dual

### ISSUE THREAD

**Issue by tonipalmgren** [cleanflight/cleanflight#3047]
FlySky IBUS and FrSky SmartPort telemetry do not work on an SP Racing F7 Dual.

Software used:
-Cleanflight (v2.2.0 and v2.3.2)
-Betaflight (unofficial, nightly build, since there is no official release yet for the SP Racing F7 Dual)

What I have tried:
-using the dedicated receiver port on the FC; R2 for SBUS, T5/UART5 for the telemetry (as recommended in the manual), also UART4 and UART1 for the telemetry
-discovering new sensors on the Taranis with the quad armed, motors running
-using FlySky FS-I6X transmitter and FlySky FS-IA6B receiver
-using FrSky Taranis X9D+ transmitter and FrSky X8R receiver
-using another X8R receiver
-upgrading Taranis with the latest OpenTX
-upgrading X8R with the latest firmware
-"set tlm_inversion", "set frsky_inversion" and "set telemetry_inversion" commands, none of which is recognized by the CLI
-googling

I would had tried softserial, but it isn't available on any firmware yet.

**Comment by Alofly** [comment]
Hi, I have the same trouble with de FrSky R-XSR on CleanFlight 2.2.0 and 2.3.2

**Comment by dydek** [comment]
I was trying to set up it too - using UART2 with IBUS telemetry configured without any luck ( cleanflight  2.3.2 , FlySky iA6b receiver and FS-I6 transmitter )

**Comment by tonipalmgren** [comment]
Ok. Good to know. I've updated every firmware and tried everything that has come to my mind. Unfortunately I've already ordered another SP Racing F7 Dual and it's too late to cancel. I hope this issue will be resolved quickly. I've got one build and another repair on hold until this gets resolved, as it's total insanity to fly without a low voltage warning.

**Comment by micworg** [comment]
Same issue here .. hopefully this can be fixed soon.

**Comment by tonipalmgren** [comment]
@micworg What hardware were you using? Flight controller, receiver, and transmitter?

**Comment by hydra** [comment]
Seems rather odd, I'll check it soon - I fly FrSky with XSR's most of the time and telemetry has always worked for me on the SPRacingF7DUAL.

**Comment by hydra** [comment]
confirmed X8R + Taranis Open Tx 2.1.7 works with SBUS on UART2, SmartPort on UART5 with the following setting diff:
```
# Cleanflight / SPRACINGF7DUAL (SP7D) 2.3.2 May 15 2018 / 15:54:03 (208d4cce4) MSP API: 1.38
set tlm_inverted = OFF
```

**Comment by Alofly** [comment]
I have configured my F7 in the same way I did in my F3NEO and F4EVO. 
But on the F7 it does not work. What would I have to do differently in F7 to make it work?

If there really is nothing different to do ... What would I have to do to make you believe what I say?

**Comment by tonipalmgren** [comment]
@hydra "tlm_inverted = off" seems to have "fixed" it. I hadn't found this command, but hey, I only spent hours and hours searching on Google and watching videos. :P I saw you create a Wiki-type page for the S7 Dual. Could that be a nice place for this kind of info? A place to keep all the F7 Dual related stuff in, regardless of other hardware used. Would had saved a lot of headache... I was also under the impression that turning on SmartPort on some port would automatically turn invertion off, if needed, which is partly why I stopped googling after only a gazillion hours... :P

So, to sum it up, "set tlm_inverted = off" is needed with Taranis X9D+ and X8R to get the telemetry to work. I'm moving from FlySky to FrSky, so I won't test FlySky telemetry for now, but maybe later.

@Alofly All I did was "set tlm_inverted = off" in the Cleanflight CLI, then typed "save" and everything worked. Though I have the X8R, not the XSR.

**Comment by micworg** [comment]
"tlm_inverted = off" fixed it for me too.

**Comment by tonipalmgren** [comment]
@Alofly I suggest you try the "set tlm_inverted = off" in the command line. This seems to work, at least for the FrSky products.

**Comment by tonipalmgren** [comment]
"set tlm_inverted = off" fixes this on the FrSky products.

**Comment by Alofly** [comment]
tonipalmgren, you're right. I tried it before but I did not put the "save" command at the end

Thank you so much for your patience
